src/index.html

```diff
@@ -1,11 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9c25278351.js"crossorigin="anonymous"></script>
+  <title><%= htmlWebpackPlugin.options.title %></title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+  <header>
+  <h1><%= htmlWebpackPlugin.options.title %></h1>
+  </header>
+  <section>
+  <ul class="todo-list">
+    <li>Today's TO DO</li>
+    <li class='list-item'><input class='item-input' id="desc-input" placeholder='Add to your list ...'/></li>
+  </ul>
```

Comment:
- [ ] When I run npm start I should see the styled list that you have created dynamically as below: 
![image](https://user-images.githubusercontent.com/91486791/206261577-ca1da09e-04a9-4659-ad53-f891128c3c0a.png)
your index page seems like this: 
![image](https://user-images.githubusercontent.com/91486791/206261700-bcbe72c2-5efd-46c1-ab5c-11f514cc041e.png)

Reply:
@tugsem 
Thank you so much for the review.

I have kindly fixed the above issue at [this link](https://github.com/Lucash2022/To_Do_List/commit/44c1c147705bff74452908e1aa903d214ed648cc)